moped-database/migrations/1635521196468_create_table_public_moped_project_types/up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE "public"."moped_project_types"(
+  "project_id" integer NOT NULL,
+  "project_type_id" integer NOT NULL,
+  "date_added" timestamptz DEFAULT clock_timestamp(),
+  "added_by" integer,
+  PRIMARY KEY ("project_type_id"),
+  FOREIGN KEY ("project_type_id") REFERENCES "public"."moped_types"("type_id") ON UPDATE restrict ON DELETE restrict,
+  FOREIGN KEY ("project_id") REFERENCES "public"."moped_project"("project_id") ON UPDATE restrict ON DELETE restrict,
+  UNIQUE ("project_type_id"));
+COMMENT ON TABLE public.moped_project_types IS 'List of related project types';
+
+DROP TABLE "public"."moped_proj_categories";
+DROP TABLE "public"."moped_categories";
```

Comment:
Those should be fine 👍 , luckily there isn't any data in moped_proj_categories in neither staging or prod, so this migration should succeed.